protostar/starknet_gateway/gateway_response.py

```diff
@@ -22,6 +24,6 @@ class SuccessfulInvokeResponse:
 
 @dataclass
 class SuccessfulDeployAccountResponse:
-    address: int
+    address: AccountAddress
```

Comment:
Regular Address is fine here. We should accept lowest-possible class in the hierarchy

Reply:
> We should accept lowest-possible class in the hierarchy

I don't get it tbh. We should use the most specific available types.

Reply:
Replaced AccountAddress with Address